Dockerfile

```diff
@@ -19,10 +10,11 @@ ARG GITHUB_TOKEN
 RUN mkdir -p /mnt/vol
 COPY rplatform/dependencies.yaml rplatform/.github_access_token.txt* /mnt/vol
 RUN echo "$GITHUB_TOKEN" >> /mnt/vol/.github_access_token.txt
+RUN Rscript -e 'BiocManager::install(c("BiocStyle", "qs"))'
```

Comment:
packages in the Suggests that are essential to build vignette (which happens within `pkgdown::build_site`.